miden-crypto/benches/README.md

```diff
@@ -105,3 +105,74 @@ cargo run --no-default-features --features=executable,hashmaps
 ```
 
 The benchmark parameters may also be customized with the `-s`/`--size`, `-i`/`--insertions`, and `-u`/`--updates` options.
+
+## Configuration
```

Comment:
Maybe it is worth moving this section (as well as "Instructions" section to the top of the file? And maybe adding an intro paragraph explaining things at the higher level.

Or maybe we should split the file into two high level sections: explanations about the benchmarks (e.g., how to run/configure/add benchmark) and benchmark results which (i.e., what we have already before the "instructions" section).